### PR TITLE
Feature/exchange rate

### PR DESCRIPTION
**Pull request by AliAlperenSonmez**
Exchange rate API is added. Note that I needed to add 2 extra lines to backend/config/urls.py file.

### CODE REVIEW

**Comment by enfurars** [comment]
I have checked the 9 files changed with this branch and did not see any problem.

**Review by enfurars — COMMENTED** [review]
I could not understand few changes so leaving those to other reviewers.

File: practice-app/backend/api/viewsets/exchange_rate_viewset.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+from rest_framework.decorators import action
+from rest_framework.response import Response
+
+
+class ExchangeRateViewSet(viewsets.ModelViewSet):
+    queryset = ExchangeRate.objects.all()
+    serializer_class = ExchangeRateSerializer
+    filterset_fields = ('from_currency' , 'to_currency')
+
+    @action(methods=['get'],detail=False)
```
Comment (EmreBatuhan) [inline practice-app/backend/api/viewsets/exchange_rate_viewset.py:13]: Bu data kısmı gereksiz bunu kaldırabilirsin

**Review by EmreBatuhan — APPROVED** [review]
Didn't notice a problem.

**Review by selinisik — APPROVED** [review]
LGTM!

File: practice-app/backend/db.sqlite3
Comment (YigitSekerci) [inline practice-app/backend/db.sqlite3:1]: Please remove this file from git. Gitignore update is in the another branch that's why your code automatically tried to add it to git.

File: practice-app/backend/config/urls.py [hunk 2]
```diff
@@ -1,7 +1,10 @@
 from django.contrib import admin
-from django.urls import path,include
+from django.urls import path, include
+from django.views.generic import RedirectView # Added this line
 
 urlpatterns = [
+    path('', RedirectView.as_view(url='api/', permanent=False)), # Added this line
```
Comment (YigitSekerci) [inline practice-app/backend/config/urls.py:6]: burası ne yapıyor?

**Review by YigitSekerci — APPROVED** [review]
Looks solid, just have few comments

File: practice-app/backend/api/migrations/0002_exchangerate.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.2.1 on 2023-05-10 22:23
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0001_initial'),
```
Comment (YigitSekerci) [inline practice-app/backend/api/migrations/0002_exchangerate.py:9]: It should be empty string I guess

File: practice-app/backend/config/urls.py (hunk 2 above)
Comment (YigitSekerci) [reply practice-app/backend/config/urls.py:6]: Commentleri kaldırsan daha iyi olabilir

**Pull request merged**